CHANGELOG.md

```diff
@@ -71,6 +71,7 @@ You can find its changes [documented below](#070---2021-01-01).
 - `EventCtx::submit_notification_without_warning` ([#2141] by [@xarvic])
 - `WidgetPod::requested_layout` ([#2145] by [@xarvic])
 - Make `Parse` work better with floats and similar types ([#2148] by [@superfell])
+- Introduced intrinsics ([#2172] by [@sjoshid])
```

Comment:
Let's make this a bit more explicit and descriptive. I would suggest something like:
```
`compute_max_intrinsic` method to the `Widget` trait, which determines the maximum useful dimension of the widget ([#2172] by [@sjoshid])
```

We also need to define the `[#2172]` link at the bottom of the changelog where other issue links are defined.